receiver/zipkinreceiver/factory.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package zipkinreceiver
+
+import (
+	"context"
+
+	"github.com/open-telemetry/opentelemetry-service/consumer"
+	"github.com/open-telemetry/opentelemetry-service/internal/configmodels"
+	"github.com/open-telemetry/opentelemetry-service/internal/factories"
+	"github.com/open-telemetry/opentelemetry-service/receiver"
+)
+
+// This file implements factory for Zipkin receiver.
+
+var _ = factories.RegisterReceiverFactory(&ReceiverFactory{})
+
+const (
+	// The value of "type" key in configuration.
+	typeStr = "zipkin"
+
+	defaultBindEndpoint = "127.0.0.1:9411"
```

Comment:
Remind me: how are we going to handle the different defaults between agent and collector? I thought that we had files for that...

Reply:
I think we concluded that we will provide default config files with agent and collector distributions.